package.json

```diff
@@ -32,6 +32,7 @@
   "dependencies": {
     "arrify": "^1.0.1",
     "chalk": "^1.1.3",
+    "minimatch": "^3.0.3",
```

Comment:
good catch. Minimatch is actually a dependency of `node-glob` which doesn't export any kind of function to use here :( But I ran `yarn add minimatch`, surprisingly, I did not cause any updates of `yarn.lock`, because latest minimatch version is already installed at `./node_modules`.